declarative-pipeline-migration-assistant/src/main/webapp/js/prism-live.js

```diff
@@ -130,7 +130,7 @@ var _ = Prism.Live = class PrismLive {
 						var snippetExpanded = this.expandSnippet(selector);
 
 						if (snippetExpanded) {
-							requestAnimationFrame(() => $.fire(this.textarea, "input"));
+							requestAnimationFrame(() => this.textarea.dispatchEvent(new Event("input")));
```

Comment:
According to https://github.com/jenkinsci/declarative-pipeline-migration-assistant-plugin/blob/master/declarative-pipeline-migration-assistant/src/main/webapp/js/prism-live.js#L19 here `$` does not refer to a prototype object but a [blissfull.js](https://blissfuljs.com/docs) one. Why are you changing it? Did you test if this was generating any errors without prototype?

Reply:
Yes, the $ is referring to Bliss js instance. And yes, I have tested it without prototype. It did not create any error. Got it.